ports/psoc6/main.c

```diff
@@ -55,6 +56,31 @@ void mpy_task(void *arg);
 
 TaskHandle_t mpy_task_handle;
 
+
```

Comment:
Maybe used in the next rounds of refactor.

Reply:
Yes, If this is required, it should be out of the main. Let me remove it.